### PR TITLE
Added documentation for API rate limits

### PR DESCRIPTION
**Pull request by anero**
screenshot 2018-12-06 17 23 27

### CODE REVIEW

File: source/includes/_authenticated_api_intro.md [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
+We apply the following per-domain rate limits on HTTP requests made to the API:
+
+* **OAuth Token exchange (`/oauth/token`)**: 10 requests per minute.
+* **Authenticated REST API (`/api/v1/*`)**: 1000 requests per minute.
+
+Once the limit is reached, we will return an HTTP response with status code `429 - Too Many Requests` and the following headers:
+
+* `RateLimit-Limit`: returns the number of requests per minute allowed on the endpoint, values will match the ones listed above depending if you are invoking the _OAuth Token Exchange_ or one of the _Authenticated REST API_ endpoints.
+* `RateLimit-Remaining`: this header will always have `0` as value, since this response is only returned when the rate limit has been reached.
+* `RateLimit-Reset`: returns the number of seconds until the current rate limit is reset.
```
Comment (anero) [inline source/includes/_authenticated_api_intro.md:20]: At least that's what I think this code does (?):
```
'RateLimit-Reset' => (now + (match_data[:period] - now % match_data[:period])).to_s
```

**Review by lavaturtle — APPROVED** [review]
:+1:

**Comment by woodhull** [comment]
Could you research how long the tokens last for? That's the other part I wanted to document or change the setting for if it is super short expiry

**Comment by anero** [comment]
Does this make sense?

screenshot 2018-12-06 18 08 20

**Pull request merged**